common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -173,7 +178,11 @@ class CapaFields(object):
     student_answers = Dict(help=_("Dictionary with the current student responses"), scope=Scope.user_state)
     done = Boolean(help=_("Whether the student has answered the problem"), scope=Scope.user_state)
     seed = Integer(help=_("Random seed for this student"), scope=Scope.user_state)
-    minutes_allowed = Integer(help=_("EXPERIMENTAL FEATURE: DO NOT USE.  Number of minutes allowed to finish this assessment. Set 0 for no time-limit"),
+    minutes_allowed = IntegerWithWarningField(display_name=("Minutes Allowed"),
+                              help=_("Number of minutes allowed to finish this assessment. Set 0 for no time-limit. "
```

Comment:
The formatting looks off here. Can we place each param on its own line, per the style guide?